luigi/scheduler.py

```diff
@@ -366,13 +379,31 @@ def get_active_tasks(self, status=None):
             for task in six.itervalues(self._tasks):
                 yield task
 
+    def get_batch_running_tasks(self, batch_id):
+        if batch_id is None:
+            return []
+        else:
+            return [
+                task for task in self.get_active_tasks(BATCH_RUNNING)
+                if task.batch_id == batch_id
+            ]
+
     def get_running_tasks(self):
         return six.itervalues(self._status_tasks[RUNNING])
 
     def get_pending_tasks(self):
         return itertools.chain.from_iterable(six.itervalues(self._status_tasks[status])
                                              for status in [PENDING, RUNNING])
 
+    def set_batcher(self, worker_id, family, batcher_args, max_batch_size):
+        if max_batch_size is None:
+            max_batch_size = float('inf')
```

Comment:
Can we avoid in `None` in the code by much earlier setting this default? Perhaps as the default value for the `rpc_method`